storage/replica.go

```diff
@@ -1040,6 +1044,14 @@ func (r *Replica) addReadOnlyCmd(ctx context.Context, ba roachpb.BatchRequest) (
 	return br, pErr
 }
 
+// TODO(tschottdorf): temporary assertion for #5725.
```

Comment:
Can you elaborate a little bit on how this relates to 5725.  Just so when looking at the code in the future others don't need to open the issue to get the gist.